sql/mob_spawn_mods.sql

```diff
@@ -1561,6 +1561,12 @@ INSERT INTO `mob_spawn_mods` VALUES (17092904,1,1800,1); -- min gil 1800
 INSERT INTO `mob_spawn_mods` VALUES (17092904,2,5800,1); -- max gil 5800
 INSERT INTO `mob_spawn_mods` VALUES (17092904,15,2500,1); -- mug gil 2500
 
+-- Peg Powler
+-- Based on wiki (https://ffxiclopedia.fandom.com/wiki/Peg_Powler)
+INSERT INTO `mob_spawn_mods` VALUES (17649761,1,6000,1); -- min gil 6000
+INSERT INTO `mob_spawn_mods` VALUES (17649761,2,9200,1); -- max gil 9200
+INSERT INTO `mob_spawn_mods` VALUES (17649761,15,4500,1); -- mug gil 4500
```

Comment:
Is there proof of this being in era? I don't see gil drop until 2010

Reply:
That is a good point. I just assumed it was.

Reply:
Always best to look back in the history of the wiki entry. If it doesn't match with what's being reported, we'll have to find evidence for it elsewhere

Reply:
![1](https://github.com/AirSkyBoat/AirSkyBoat/assets/27468039/67a0510b-77f1-47d2-9aa1-4d41395469fe)
![2](https://github.com/AirSkyBoat/AirSkyBoat/assets/27468039/73a9a142-917f-45a8-bdb6-287a4bb8f157)
![3](https://github.com/AirSkyBoat/AirSkyBoat/assets/27468039/57595907-c9f2-49bf-adc7-bd10b470a8ac)
![4](https://github.com/AirSkyBoat/AirSkyBoat/assets/27468039/7423bbf9-7d2f-46c0-9b57-1b6132ea6bda)

Reply:
I'd say this is sufficient. Thank you for hunting that down!